uint32 size
float64 min_distance
geometry_msgs/Point travel_point
geometry_msgs/Point[] frontier_cells